submissions/okkkko/js_oop/app.js

```diff
@@ -1,87 +1,102 @@
-const PLAYER_START_POSUTION = {x: 200, y: 380};
+const PLAYER_START_POSITION = { x: 200, y: 380 };
 const STEP_Y = 80;
-const STEP_X = 100;
-const FIRST_ROW = 60;
-const BUGS_START_POSUTION = {x:-200, y: [FIRST_ROW, FIRST_ROW + STEP_Y, FIRST_ROW + STEP_Y*2, FIRST_ROW + STEP_Y*3]};
-const allEnemies = [];
+const STEP_X = 102;
+const BUGS_PADDING = 18;
+const BUGS_START_POSITION = {
+  x: -200,
+  y: [1, 3, 4].map((rowNumber) => rowNumber * STEP_Y - BUGS_PADDING),
```

Comment:
You didn't properly understand me. 
 
this must not be array of positions, because this variable don't want to know how many bugs there will be in game


just function which accept rowNumber and returns y position. It will be more abstract.